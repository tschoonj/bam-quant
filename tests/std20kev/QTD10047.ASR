File: C:\xrfdaten\2014KW30\chris\std20kev\std10047.spe
Start: 21.07.2014 18:24:36
Stop: 21.07.2014 18:26:17
BESSY: 298.8836856 298.8343979
IONI: 2.942642E-11 2.940332E-11
ScanMotor1 exp_x: 1 1
ScanMotor2 exp_y: 1 1
Live: 98006
Run: 100000
Dead: 2
ICR: 842
OCR: 0
Accumulated BESSY: 59692.3191552
Accumulated Ioni: 5.87099396000002E-09
N Accumulation: 200
Normfactor:      3.4758874E-8
E_cal:      0.020000000
E_cal_offset:      0.000000
Comments: Ag
exp_x 131.260406494141 131.2604
exp_y 92.3237991333008 92.3238
exp_z 53.8899993896484 53.8901
exp_w 135 135.0001
exp_Tisch 870.6 870.600001928
dcm_theta 0 0
det_y 52.85 52.85
dmm_energy 20 20
dcm_energy 23 -184976.74536672
S3_horSize 0.03 3.00000000000001E-02
S3_verSize 0.03 3.00000000000001E-02
$PEAKS:   
      1
  47   2      2.984     28120.       169.       5.24
