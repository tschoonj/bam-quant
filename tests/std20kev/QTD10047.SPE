$DATA:
0 2000
22966
14437
6260
1781
325
56
9
0
0
0
0
0
0
0
0
0
5
39
26
30
36
27
31
24
21
32
25
20
35
19
16
33
27
26
27
14
21
20
24
15
23
20
18
21
24
20
20
18
20
18
22
18
19
12
14
24
21
30
36
48
69
84
102
72
40
28
34
33
40
35
47
33
50
42
31
31
23
29
24
18
21
26
20
16
21
20
16
18
23
24
23
14
14
12
17
21
19
17
17
24
23
26
27
18
37
27
26
26
17
26
31
23
22
27
24
22
20
33
29
36
26
41
25
33
32
39
57
90
112
147
172
145
132
121
106
81
82
81
109
142
133
207
423
725
1433
2418
3444
4249
4618
3958
3061
2173
1635
1560
1843
2277
2690
2732
2579
2181
1940
1516
1201
923
798
619
537
463
317
273
170
129
98
97
99
97
79
56
50
24
23
47
60
79
80
66
89
74
54
37
32
18
16
6
11
14
7
12
13
5
3
6
5
5
5
7
6
7
6
10
11
8
4
7
6
12
6
7
10
8
3
10
4
13
5
12
2
12
10
6
4
7
17
11
7
7
9
9
11
7
5
17
9
7
7
17
17
6
11
10
12
12
18
11
5
8
10
6
7
3
14
10
9
7
13
9
10
8
11
7
8
9
9
8
8
9
4
12
5
11
9
12
9
7
10
14
13
10
8
13
12
15
11
9
15
15
12
9
9
10
8
9
9
15
14
10
13
12
13
17
16
7
16
20
11
11
16
14
16
9
16
14
9
9
14
14
16
9
7
15
11
11
11
18
4
14
10
18
14
15
13
17
14
9
16
15
21
9
16
9
17
18
15
13
14
15
20
12
14
11
13
14
12
17
15
14
12
23
14
14
12
15
9
16
18
16
16
15
15
9
10
9
18
10
15
13
17
24
18
16
21
22
15
21
29
30
28
22
28
43
23
32
24
18
22
21
17
19
13
15
16
15
5
22
20
15
11
15
18
12
18
11
12
16
14
14
29
23
21
15
17
16
23
17
23
9
13
24
14
23
23
21
14
19
16
23
14
18
25
12
15
16
15
20
22
17
14
13
18
20
13
18
17
20
20
16
15
16
24
21
25
20
14
14
19
21
18
20
26
12
24
25
21
17
11
6
15
14
17
22
14
14
15
18
21
16
27
13
19
16
18
16
11
15
22
25
24
20
23
13
20
17
16
18
20
19
25
25
21
20
16
18
17
17
24
21
22
13
26
18
21
22
20
24
16
15
25
17
25
22
8
23
17
15
21
21
23
10
17
27
20
24
24
24
21
20
16
18
20
21
17
18
11
18
26
19
14
15
13
24
15
19
21
11
13
17
24
14
19
19
14
17
17
17
23
21
28
20
25
20
25
19
27
15
17
10
12
26
18
22
24
31
17
20
11
24
13
14
19
14
17
28
17
16
14
16
19
10
19
22
23
12
16
11
26
19
12
11
26
17
16
23
17
16
12
23
14
21
15
27
19
23
15
14
24
17
17
25
25
17
25
16
19
22
17
30
27
20
17
12
12
15
23
20
13
19
12
14
12
20
16
12
26
12
23
20
22
13
15
16
22
15
18
18
17
11
22
21
21
19
12
17
17
20
24
13
25
16
15
19
19
22
19
17
29
13
24
22
30
21
20
13
22
19
19
21
16
15
18
14
18
16
19
30
16
21
16
14
21
18
12
19
30
17
26
28
17
17
24
20
20
15
21
26
19
20
19
25
14
28
14
22
24
22
22
19
22
18
34
17
25
18
28
28
31
25
14
33
34
33
37
40
42
48
56
71
95
87
99
104
94
98
97
84
79
58
62
58
40
51
26
25
28
26
30
25
33
25
26
35
23
32
24
23
31
29
21
17
34
32
23
44
20
26
34
19
20
30
22
19
21
19
16
14
15
24
25
12
12
9
9
9
15
10
4
11
5
10
7
7
8
13
5
8
5
7
3
5
9
3
5
10
5
8
8
3
10
9
4
7
7
16
8
15
10
11
7
12
9
16
16
16
8
23
12
10
20
18
12
11
23
14
13
14
6
6
14
14
15
10
13
11
18
11
11
10
16
9
9
11
14
11
16
12
15
14
16
11
14
17
13
19
11
16
16
13
16
13
19
20
17
25
21
26
21
22
20
22
22
21
22
26
26
25
21
25
34
31
30
32
33
46
28
28
24
32
43
45
43
36
48
50
59
46
42
55
52
47
41
61
47
55
54
53
60
50
45
58
72
52
80
73
85
91
89
99
98
139
148
153
189
222
241
288
291
353
357
342
359
366
409
342
405
351
346
363
308
292
253
230
238
164
172
128
122
91
73
59
40
31
28
17
17
10
17
4
5
1
3
0
2
2
0
0
0
0
0
1
0
0
2
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
2
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
1
0
0
0
0
0
1
0
1
0
0
0
0
0
1
1
0
3
4
0
3
6
3
5
7
14
8
9
10
14
9
14
11
10
9
11
14
4
2
5
1
4
1
0
0
1
0
0
0
1
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
1
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
1
0
0
1
0
0
0
0
1
4
1
2
3
6
0
1
1
0
0
2
0
2
1
0
1
0
0
0
0
0
0
0
1
0
0
2
0
0
1
1
0
0
0
3
0
1
1
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
File: C:\xrfdaten\2014KW30\chris\std20kev\std10047.spe
Start: 21.07.2014 18:24:36
Stop: 21.07.2014 18:26:17
BESSY: 298.8836856 298.8343979
IONI: 2.942642E-11 2.940332E-11
ScanMotor1 exp_x: 1 1
ScanMotor2 exp_y: 1 1
Live: 98006
Run: 100000
Dead: 2
ICR: 842
OCR: 0
Accumulated BESSY: 59692.3191552
Accumulated Ioni: 5.87099396000002E-09
N Accumulation: 200
Normfactor:      0.34758874
E_cal:      0.020000000
E_cal_offset:      0.000000
Comments: Ag
 
exp_x 131.260406494141 131.2604
exp_y 92.3237991333008 92.3238
exp_z 53.8899993896484 53.8901
exp_w 135 135.0001
exp_Tisch 870.6 870.600001928
dcm_theta 0 0
det_y 52.85 52.85
dmm_energy 20 20
dcm_energy 23 -184976.74536672
S3_horSize 0.03 3.00000000000001E-02
S3_verSize 0.03 3.00000000000001E-02
$PEAKS:   
      1
  47   2      2.984     28120.       169.       5.24
