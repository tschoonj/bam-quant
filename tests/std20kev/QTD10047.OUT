     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ  AXIL IBM-PC V3.00                       10-08-2014   14:23:49  บ   
     บ  Spectrum: QTD10047.SPE                                     0.s บ   
     ศอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออผ   
     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ  Fitting Region: channels 100 -1980        3 iterations done    บ   
     บ  ChiSquare =    12.1   last change =   .04%    lambda= 1.E-05   บ   
     ฬอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออน   
     บ                        CALIBRATION  DATA                        บ   
     ฬออออออออออออออออหออออออออออออออออออออออออหอออออออออออออออออออออออน   
     บ                บ   Initial estimate     บ    Final estimate     บ   
     ฬออออออออออออออออฮออออออออออออออออออออออออฮอออออออออออออออออออออออน   
     บ  ZERO (eV)     บ     -101.1 ๑   100.0   บ    -101.1 ๑     5.4   บ   
     บ  GAIN (eV/ch)  บ     20.735 ๑   2.000   บ    20.735 ๑    .037   บ   
     บ det NOISE (eV) บ       44.8 ๑    40.0   บ      44.8 ๑    31.0   บ   
     บ  FANO factor   บ       .208 ๑    .050   บ      .208 ๑    .030   บ   
     ศออออออออออออออออสออออออออออออออออออออออออสอออออออออออออออออออออออผ   
     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ                           PEAK  DATA                            บ   
     ฬอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออน   
     บ  # Line       E(KeV)  rel. int.   peak area   st. dev    chi-sq บ   
     บ               chan#   fwhm (eV)      backgr            tot. abs บ   
     ฬอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออน   
     บ                                                                 บ   
     บ  1 Ag-L1                              5921. ๑    114.           บ   
     บ    L1M3        3.234    .50249        2975. ๑     58.    102.2  บ   
     บ              160.842    127.83         120.           5.29E-01  บ   
     บ    L1M2        3.203    .29798        1764. ๑     36.    101.1  บ   
     บ              159.347    127.29         121.           5.19E-01  บ   
     บ    L1N3        3.747    .19079        1130. ๑     24.     66.3  บ   
     บ              185.582    136.43         113.           6.57E-01  บ   
     บ    L1M5        3.437    .00874          52. ๑     11.     75.0  บ   
     บ              170.632    131.30         118.           5.83E-01  บ   
     บ                                                                 บ   
     บ  2 Ag-L2                             14009. ๑    155.           บ   
     บ    L2M4        3.151    .86735       12151. ๑    135.     69.5  บ   
     บ              156.839    126.38         121.           5.10E-01  บ   
     บ    L2N4        3.520    .10722        1502. ๑     20.     97.8  บ   
     บ              174.635    132.70         116.           6.04E-01  บ   
     บ    L2M1        2.806    .01981         278. ๑     11.     19.8  บ   
     บ              140.201    120.18         105.           3.93E-01  บ   
     บ    L2N1        3.428    .00563          79. ๑     11.     75.0  บ   
     บ              170.198    131.15         118.           5.80E-01  บ   
     บ                                                                 บ   
     บ  3 Ag-L3                             32685. ๑    196.           บ   
     บ    L3M5        2.984    .86034       28120. ๑    169.      5.2  บ   
     บ              148.785    123.42         124.           4.58E-01  บ   
     บ    L3N5        3.348    .10933        3574. ๑     24.     73.5  บ   
     บ              166.340    129.79         119.           5.62E-01  บ   
     บ    L3M1        2.634    .02306         754. ๑     11.     13.8  บ   
     บ              131.906    116.97         107.           3.32E-01  บ   
     บ    L3N1        3.256    .00727         238. ๑     11.    102.1  บ   
     บ              161.903    128.21         120.           5.35E-01  บ   
     บ                                                                 บ   
     บ  4 Pile_up                              73. ๑     15.           บ   
     บ    L2M1        6.156    .54475          40. ๑     13.       .9  บ   
     บ              301.775    171.15         113.           9.08E-01  บ   
     บ    L2N1        5.968    .45525          33. ๑     12.       .9  บ   
     บ              292.694    168.70         101.           8.99E-01  บ   
     บ                                                                 บ   
     ศอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออผ   
                                                                           
     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ                         FILTER BACKGROUND                       บ   
     ศอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออผ   
                                                                           
